### ISSUE TITLE
Consider Kanban Use for GLA6 Volunteers

### ISSUE THREAD

**Issue by emcglackenr1** [CodeYourFuture/CYF-Staff-Tickets#678]
**onsider Kanban Use for GLA6 Volunteers**

We previously used the Kanban for volunteer meetings at the start of the cohort. Because it aligns with ways of working for developers, and is how CYF staff and other regions organise work, it would be great to reintroduce this. It might help with engagement and organisation. 
Previous feedback was that it was structured in a non-traditional way, and was stressful to use, but that volunteers liked typical Kanbans.

**Relevant information:**
Volunteer meeting is organised in channel #scotland-volunteers. See this channel or the Scotland Google Calendar for details on when the meeting is happening. Agenda points for the meeting go in the doc linked in the meeting workflow on #scotland-volunteers. 

Acceptance Criteria - Kanban

- [x]  Tidy up the current one e.g. archiving and closing old issues that are no longer relevant and/or have been completed - Dara
- [x]  Highlight the kanban as a way to organise work in the module - get feedback
- [x]  Ensure active volunteers are on the github / know where to look for the kanban
- [x]  Show to trainees / course completors

**Comment by haldenb** [comment]
Is the Kanban something that trainees/ volunteers want? If so, what are the blockers? If they see the value in it, why aren't they using it? Have volunteers asked us to do this?

I am struggling to see what I can do to help people who use Kanban professionally when I am a novice.

**Comment by emcglackenr1** [comment]
It was very messy so I have tidied it up. No one wanted to use it because it wasn't organised like an actual Kanban. So, it would basically just be about mentioning it to them and see if they want to use it or not now I have addressed their blockers with it. 

This ties into [this post though](https://codeyourfuture.slack.com/archives/G6H0HAY10/p1710246265805389), where I thought this ticket could also be handed over to volunteers, but it might not be as easy to integrate as I thought. So, up to you don't think discussing it is part of your role.
